src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -12,7 +12,8 @@ export interface ParamsForProvider {
 
 export default class PaypalCommerceRequestSender {
     constructor(
-        private _requestSender: RequestSender
+        private _requestSender: RequestSender,
+        private isAPM?: boolean | undefined
```

Comment:
Can you prefix private properties with an underscore?

Reply:
fixed